packages/stores/src/account/index.ts

```diff
@@ -815,6 +815,91 @@ export class OsmosisAccountImpl {
     );
   }
 
+  async sendAddToConcentratedLiquidityPositionMsg(
+    positionId: string,
+    amount0: string,
+    amount1: string,
+    maxSlippage = "2.5",
+    memo: string = "",
+    onFulfill?: (tx: any) => void
+  ) {
+    // refresh position
+    const queryPosition =
+      this.queries.queryLiquidityPositionsById.getForPositionId(positionId);
+    await queryPosition.waitFreshResponse();
```

Comment:
do we need to worry about errors in these functions? We aren't handling error states

Reply:
Eh yes that's possible, if the endpoint fails. But generally we assume queries will work as expected, and errors logged in those base query classes, otherwise we'd have to write a lot of boilerplate to handle errors across every use.